db/models.js

```diff
@@ -0,0 +1,109 @@
+const client = require('./index');
+const helpers = require('./helpers');
+
+const getReviews = (productId, sort, offset, count) => {
+  let sortValue;
+  if (sort === 'newest') {
+    sortValue = 'date DESC';
+  } else if (sort === 'helpful') {
+    sortValue = 'helpfulness DESC';
+  } else if (sort === 'relevant') {
+    sortValue = 'date DESC, helpfulness DESC';
+  } else { console.log('invalid sort value'); }
+
+  const query = {
+    text: `SELECT * FROM reviews WHERE product_id = $1 AND reported = false ORDER BY ${sortValue} OFFSET $2 LIMIT $3`,
```

Comment:
@jabrake : Shouldn't sortValue be also parameterized like the other dynamic variables such as productId and sort?

Reply:
Good catch - updated!